src/dashboard/dashboard/settings.py

```diff
@@ -74,12 +79,12 @@
 # https://docs.djangoproject.com/en/5.1/ref/settings/#databases
 DATABASES = {
     "default": {
-        "ENGINE": os.environ.get("DASHBOARD_DB_ENGINE"),
-        "NAME": os.environ.get("DASHBOARD_DB_NAME"),
-        "USER": os.environ.get("DASHBOARD_DB_USER"),
-        "PASSWORD": os.environ.get("DASHBOARD_DB_PASSWORD"),
-        "HOST": os.environ.get("DASHBOARD_DB_HOST"),
-        "PORT": os.environ.get("DASHBOARD_DB_PORT"),
+        "ENGINE": env.str("DB_ENGINE"),
+        "NAME": env.str("DB_NAME"),
+        "USER": env.str("DB_USER"),
+        "PASSWORD": env.str("DB_PASSWORD"),
+        "HOST": env.str("DB_HOST"),
+        "PORT": env.str("DB_PORT"),
```

Comment:
There is a special method (_i.e._ `env.db()`) for database URL, see: https://django-environ.readthedocs.io/en/latest/quickstart.html